### PR TITLE
Make use of awesome-client commands

### PR DESCRIPTION
**Pull request by psychon**
This uses the new support introduced in f0f31bc30588 / https://github.com/awesomeWM/awesome/pull/1058 in the docs and in tests/run.sh, removing an useless use of cat/echo.

### CODE REVIEW

**Comment by psychon** [comment]
...and now this also fixes the return value for things like `awesome-client "error('foo')"`!

File: utils/awesome-client [hunk 1]
```diff
@@ -44,14 +44,16 @@ a_dbus_send()
     $DBUS_SEND --dest=$DBUS_DEST --type=method_call --print-reply \
```
Comment (blueyed) [inline utils/awesome-client]: commit message: with "when sending a command that it for from stdin failed": s/for/got/ ?

Comment (psychon) [inline utils/awesome-client]: Yup, done.

**Comment by Elv13** [comment]
👍

**Comment by actionless** [comment]
:+1:

**Pull request merged**